test/core/test_compute.py

```diff
@@ -188,3 +224,54 @@ def _assert_inspect(self,
         self.assertEqual(exp_kwonlyargs, argspec[4], msg='kwonlyargs')
         self.assertEqual(exp_kwonlydefaults, argspec[5], msg='kwonlydefaults')
         self.assertEqual(exp_annotations, argspec[6], msg='annotations')
+
+    def test_xarray_apply_with_dim_reduction(self):
```

Comment:
What is the meaning of this and the following test? They seem to test xarray rather than xcube.

Reply:
It not only _seems_, it does. As the name says, it ensures that xarray behaves as it is expected by our impl. under test.